src/10-webpage-publishing-with-ipfs/index.md

```diff
@@ -0,0 +1,308 @@
+---
+date: 2016-02-23
+id: 10-webpage-publishing-with-ipfs
+template: tmpl/layouts/post.html
+baseurl: ..
+breadcrumbs:
+  - {name: "10-webpage-publishing-with-ipfs", link: "./" }
+tags: publishing, web, app, publish
+title: Static Web publishing, simple and distributed with IPFS
+author: David Dias
+collection: posts
+---
+
+> - Don't you hate it when your links break?
+> - Don't you hate it when your web host gets attacked and your websites go down?
+> - Don't you hate managing servers just to self-host a web page?
+> - Don't you hate it when you switch cloud providers and all the data links change?
+> - Don't you hate it when people relying on your content get broken embeds?
+
+> NOTE: I think it'd be better to start stronger than rhetorical questions. For instance:    
```

Comment:
We're going to have to go with one of these, or the other. The questions, or the paragraph.